src/test/java/guitests/DialogMessageTests.java

```diff
@@ -38,4 +38,12 @@ public void showYesNoWarningDialogTest() throws ExecutionException, InterruptedE
         click("nO");
         assertEquals(false, noTask.get());
     }
+
+    @Test
+    public void showInformationDialogTest() {
+        Platform.runLater(() -> DialogMessage.showInformationDialog("Information", "Test Information Dialog"));
+        sleep(1000);
```

Comment:
I was thinking with `awaitCondition`, but sure, that sounds like a better idea. Yeah we just want to synchronise as much as possible.